routers/cart-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.use(bodyParser.json({limit: '5000kb'}))
+const cors = require('cors')
+CartRouter.use(cors())
```

Comment:
I think doing this `route.use(cors())` for every route is not a good thing because you need to do this for every single route.
Just Remove this and add this in your index.js file as `app.use(cors())`

Reply:
worked on it and moved to index.js in all places